e2e_test/sink/mqtt_sink.slt

```diff
@@ -0,0 +1,66 @@
+statement ok
+CREATE TABLE mqtt (
+  device_id varchar,
+  temperature double,
+  topic varchar as '/device/' || device_id
+);
+
+statement ok
+CREATE SINK mqtt_sink
+FROM
+  mqtt
+WITH
+  (
+    connector='mqtt',
+    url ='tcp://mqtt-server',
+    topic = 'not_used',
+    topic.field = 'topic',
+    type = 'append-only',
+    retain = 'true',
+    qos = 'at_least_once',
+  ) FORMAT PLAIN ENCODE JSON (
+      force_append_only='true',
+  );
+
+statement ok
+CREATE TABLE mqtt_source
+(
+  device_id varchar,
+  temperature double
+)
+WITH (
+    connector='mqtt',
+    url='tcp://mqtt-server',
+    topic= '/device/+',
+    qos = 'at_least_once',
+) FORMAT PLAIN ENCODE JSON;
+
+
+statement ok
+INSERT INTO mqtt (device_id, temperature)
+VALUES ( '12',  56.0 );
+
+statement ok
+INSERT INTO mqtt (device_id, temperature)
+VALUES ( '12',  59.0 );
+
+statement ok
+INSERT INTO mqtt (device_id, temperature)
+VALUES ( '13',  20.0 );
+
+statement ok
+INSERT INTO mqtt (device_id, temperature)
+VALUES ( '13',  22.0 );
+
+statement ok
+FLUSH;
+
+sleep 5s
+
+query IT rowsort
+SELECT * FROM pulsar_source;
```

Comment:
`pulsar_source`, do you mean `mqtt_source`?

Reply:
Thanks